ui/widgets/nuxeo-selectivity.js

```diff
@@ -7351,6 +7351,12 @@ typedArrayTags[weakMapTag] = false;
     _readonlyChanged() {
       if (this._selectivity) {
         this._selectivity.setOptions({ readOnly: this.readonly });
+        if (this.readonly) {
+          this._selectivity.setOptions({ placeholder: "" });
+        }
+        if (!this.readonly) {
+          this._selectivity.setOptions({ placeholder: this.placeholder });
+        }
```

Comment:
Maybe this works just as well?
```suggestion
        this._selectivity.setOptions({
          readOnly: this.readonly,
          placeholder: this.readonly ? "" : this.placeholder,
        });
```

Reply:
Much better. My question here is more if this logic should be in this method.